setup.cfg

```diff
@@ -47,6 +47,7 @@ zip_safe = False
 include_package_data = True
 
 install_requires =
+  aiohappyeyeballs >= 2.0.0
```

Comment:
Should we maybe put this under speedups and make it optional?

Reply:
I thought about this from the perspective that IPv6 support should be a base feature, and without Happy Eyeballs, IPV6 tends to get turned off because it cannot establish a connection before the timeout with many network setups.

At least from my experience with Home Assistant, we have IPv6 turned off in many places by setting the address family to AF_INET only because the lack of Happy Eyeballs makes using IPv6 untenable.

Reply:
OK, seems reasonable then. Hopefully we can get it into cpython in the next couple of releases and then drop the dependency again.